int64 tensions
int64 number_motors
---
bool result